### PR TITLE
Update reduce-blast-radius.mdx

### PR DESCRIPTION
**Pull request by programmerq**
Clarify behavior around MFA devices and SSO users.

### CODE REVIEW

File: docs/pages/management/security/reduce-blast-radius.mdx [hunk 1]
```diff
@@ -76,7 +76,7 @@ Choose `on` if you would like to require MFA for all users while letting them ch
 - Adjusting the Teleport signup page so a user must enroll an MFA device of the kind you have selected. If the value of `second_factor` is `on`, users will have the option to select from multiple device types.
 - Presenting the user with an MFA challenge when they run `tsh login`.
 
-If you have enabled SSO in your Teleport environment, Teleport will conduct its own MFA challenge before sending a request to the identity provider. This means that, even if an attacker has managed to compromise your organization's SSO provider (e.g., by impersonating an administrator), a malicious user will still need to authenticate using valid credentials.
+If you have enabled SSO in your Teleport environment, mfa challenges at login will only apply to Teleport local users. The IDP will be responsible for implementing any second factor challenges at login time.
```
Comment (zmb3) [inline docs/pages/management/security/reduce-blast-radius.mdx:79]:
```suggestion
If you have enabled SSO in your Teleport environment, MFA challenges at login will only apply to Teleport local users. The IdP will be responsible for implementing any second factor challenges at login time.
```

MFA is an acronym and should definitely be all caps.

For IdP/IDP, we are a bit inconsistent, but IdP is more common:

```
teleport/docs on zmb3/upload-err-stacktrace [$]
➜ ag -c IDP
cspell.json:1
pages/access-controls/idps/saml-reference.mdx:1
pages/access-controls/sso.mdx:1
pages/reference/cli.mdx:2
pages/reference/terraform-provider.mdx:6

teleport/docs on zmb3/upload-err-stacktrace [$]
➜ ag -c IdP
config.json:1
img/server-access/getting-started-diagram.svg:1
img/IBM/IBM_HA.svg:1
pages/access-controls/introduction.mdx:3
pages/access-controls/sso/oidc.mdx:1
pages/access-controls/sso/okta.mdx:7
pages/access-controls/idps/saml-guide.mdx:3
pages/access-controls/idps/saml-reference.mdx:1
pages/access-controls/idps/saml-grafana.mdx:3
pages/access-controls/login-rules.mdx:5
pages/access-controls/sso.mdx:1
pages/desktop-access/rbac.mdx:1
pages/includes/sso/idp-initiated.mdx:3
pages/connect-your-client/introduction.mdx:1
pages/server-access/guides/ssh-pam.mdx:1
pages/reference/cli.mdx:7
pages/reference/resources.mdx:3
pages/reference/terraform-provider.mdx:9
```

Lastly, I might suggest rewording this a bit. _Teleport_ MFA challenges at login only apply for local users, but the IdP may (likely will) perform its own MFA checks.

Comment (programmerq) [reply docs/pages/management/security/reduce-blast-radius.mdx]: I incorporated those suggestions!

**Review by ptgott — APPROVED** [review]

File: docs/pages/management/security/reduce-blast-radius.mdx [hunk 2]
```diff
@@ -76,7 +76,7 @@ Choose `on` if you would like to require MFA for all users while letting them ch
 - Adjusting the Teleport signup page so a user must enroll an MFA device of the kind you have selected. If the value of `second_factor` is `on`, users will have the option to select from multiple device types.
 - Presenting the user with an MFA challenge when they run `tsh login`.
 
-If you have enabled SSO in your Teleport environment, Teleport will conduct its own MFA challenge before sending a request to the identity provider. This means that, even if an attacker has managed to compromise your organization's SSO provider (e.g., by impersonating an administrator), a malicious user will still need to authenticate using valid credentials.
+If you have enabled SSO in your Teleport environment, MFA challenges at login only apply for local users, but the Idp may apply its own MFA checks.
```
Comment (zmb3) [inline docs/pages/management/security/reduce-blast-radius.mdx:79]:
```suggestion
If you have enabled SSO in your Teleport environment, MFA challenges at login only apply for local users, but the IdP may apply its own MFA checks.
```

**Review by zmb3 — APPROVED** [review]
